tinkerbell/rufio/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.0
```

Comment:
This is incidentally the same as the appVersion. There's no change to the chart and no breaking changes in the image.